### PR TITLE
test(split-button/step-sequence/step-sequence-item): apply new url approach

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

Apply new `url` approach in tests for:
`splitButton` (timing: was `02:13` -> `00:49`),
`stepSequence` (timing: was  `00:12` -> `00:05`),
`stepSequenceItem` (timing  `01:25` -> `00:51`).

### Current behaviour

Currently we are passing all parameters via `knobs`.

### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

File: cypress/fixtures/commonComponents/splitButton.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "iconType": "warning",
+    "iconPosition": "after"
+  },
+  "iconPositionBefore": {
+    "has icon": true,
+    "iconType": "warning",
+    "iconPosition": "before"
+  },
+  "default": {
+
```
Comment (KatarzynaQA) [inline cypress/fixtures/commonComponents/splitButton.json:54]: delete empty line

File: cypress/fixtures/commonComponents/stepSequence.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  "ariaLabelSpecialCharacter": {
+    "ariaLabel": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "childrenOtherLanguage": {
+    "children": "mp150ú¿¡üßä"
+  },
+  "childrenSpecialCharacter": {
+    "children": "!@#$%^*()_+-=~[];:.,?{}"
+  }
+
```
Comment (KatarzynaQA) [inline cypress/fixtures/commonComponents/stepSequence.json:52]: 3 empty lines, delete

File: cypress/support/step-definitions/split-button-steps.js [hunk 3]
```diff
@@ -123,6 +128,14 @@ When('I click {string} element of Split Button component', (element) => {
   }
 });
 
+When('I click {string} element of Split Button component in IFrame', (element) => {
+  if (element === 'first' || element === 'second' || element === 'third') {
+    additionalButtonIFrame(positionOfElement(element)).click();
+  } else {
```
Comment (KatarzynaQA) [inline cypress/support/step-definitions/split-button-steps.js:134]: use `else if`

File: cypress/fixtures/commonComponents/splitButton.json (hunk 1 above)
Comment (DlgSHi) [reply cypress/fixtures/commonComponents/splitButton.json]: deleted

File: cypress/fixtures/commonComponents/stepSequence.json (hunk 2 above)
Comment (DlgSHi) [reply cypress/fixtures/commonComponents/stepSequence.json]: deleted

File: cypress/support/step-definitions/split-button-steps.js (hunk 3 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/split-button-steps.js]: added

**Review by KatarzynaQA — APPROVED** [review]

**Pull request merged**